<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HtmlEditor</class>
 <widget class="QWidget" name="HtmlEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTML Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QMenuBar" name="menubar">
     <widget class="QMenu" name="menu_Edit">
      <property name="title">
       <string>&amp;Edit</string>
      </property>
      <addaction name="actionEditUndo"/>
      <addaction name="actionEditRedo"/>
      <addaction name="separator"/>
      <addaction name="actionEditCut"/>
      <addaction name="actionEditCopy"/>
      <addaction name="actionEditPaste"/>
      <addaction name="separator"/>
      <addaction name="actionEditSelectAll"/>
      <addaction name="separator"/>
      <addaction name="actionInsertImage"/>
      <addaction name="actionInsertHtml"/>
     </widget>
     <widget class="QMenu" name="menuF_ormat">
      <property name="title">
       <string>F&amp;ormat</string>
      </property>
      <widget class="QMenu" name="menuSt_yle">
       <property name="title">
        <string>St&amp;yle</string>
       </property>
       <addaction name="actionStyleParagraph"/>
       <addaction name="actionStyleHeading1"/>
       <addaction name="actionStyleHeading2"/>
       <addaction name="actionStyleHeading3"/>
       <addaction name="actionStyleHeading4"/>
       <addaction name="actionStyleHeading5"/>
       <addaction name="actionStyleHeading6"/>
       <addaction name="actionStylePreformatted"/>
       <addaction name="actionStyleAddress"/>
      </widget>
      <widget class="QMenu" name="menu_Align">
       <property name="title">
        <string>&amp;Align</string>
       </property>
       <addaction name="actionFormatAlignLeft"/>
       <addaction name="actionFormatAlignCenter"/>
       <addaction name="actionFormatAlignRight"/>
       <addaction name="actionFormatAlignJustify"/>
      </widget>
      <addaction name="menuSt_yle"/>
      <addaction name="menu_Align"/>
      <addaction name="separator"/>
      <addaction name="actionFormatBold"/>
      <addaction name="actionFormatItalic"/>
      <addaction name="actionFormatUnderline"/>
      <addaction name="actionFormatStrikethrough"/>
      <addaction name="separator"/>
      <addaction name="actionFormatIncreaseIndent"/>
      <addaction name="actionFormatDecreaseIndent"/>
      <addaction name="separator"/>
      <addaction name="actionFormatNumberedList"/>
      <addaction name="actionFormatBulletedList"/>
      <addaction name="separator"/>
      <addaction name="actionFormatFontName"/>
      <addaction name="actionFormatFontSize"/>
      <addaction name="separator"/>
      <addaction name="actionFormatTextColor"/>
      <addaction name="actionFormatBackgroundColor"/>
      <addaction name="actionTextOutline"/>
      <addaction name="actionTextShadow"/>
     </widget>
     <widget class="QMenu" name="menuFile">
      <property name="title">
       <string>&amp;File</string>
      </property>
      <addaction name="actionFileSave"/>
      <addaction name="separator"/>
      <addaction name="actionExit"/>
     </widget>
     <addaction name="menuFile"/>
     <addaction name="menu_Edit"/>
     <addaction name="menuF_ormat"/>
    </widget>
   </item>
   <item>
    <widget class="QToolBar" name="standardToolBar">
     <property name="windowTitle">
      <string>Standard</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <addaction name="actionFileSave"/>
     <addaction name="separator"/>
     <addaction name="actionEditUndo"/>
     <addaction name="actionEditRedo"/>
     <addaction name="separator"/>
     <addaction name="actionEditCut"/>
     <addaction name="actionEditCopy"/>
     <addaction name="actionEditPaste"/>
     <addaction name="actionFormatBold"/>
     <addaction name="actionFormatItalic"/>
     <addaction name="separator"/>
     <addaction name="actionFormatAlignLeft"/>
     <addaction name="actionFormatAlignCenter"/>
     <addaction name="actionFormatAlignRight"/>
     <addaction name="separator"/>
     <addaction name="actionFormatDecreaseIndent"/>
     <addaction name="actionFormatIncreaseIndent"/>
     <addaction name="separator"/>
     <addaction name="actionFormatNumberedList"/>
     <addaction name="actionFormatBulletedList"/>
     <addaction name="separator"/>
     <addaction name="actionInsertImage"/>
     <addaction name="actionInsertHtml"/>
     <addaction name="separator"/>
     <addaction name="actionZoomOut"/>
     <addaction name="actionZoomIn"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWebView" name="webView">
           <property name="url">
            <url>
             <string>about:blank</string>
            </url>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionFileNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset theme="document-open" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/document-open.png</normaloff>:/icons/oxygen/32x32/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset theme="document-save" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/document-save.png</normaloff>:/icons/oxygen/32x32/actions/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionEditUndo">
   <property name="icon">
    <iconset theme="edit-undo" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/edit-undo.png</normaloff>:/icons/oxygen/32x32/actions/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionEditRedo">
   <property name="icon">
    <iconset theme="edit-redo" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/edit-redo.png</normaloff>:/icons/oxygen/32x32/actions/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionEditCut">
   <property name="icon">
    <iconset theme="edit-cut" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/edit-cut.png</normaloff>:/icons/oxygen/32x32/actions/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionEditCopy">
   <property name="icon">
    <iconset theme="edit-copy" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/edit-copy.png</normaloff>:/icons/oxygen/32x32/actions/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionEditPaste">
   <property name="icon">
    <iconset theme="edit-paste" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/edit-paste.png</normaloff>:/icons/oxygen/32x32/actions/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionEditSelectAll">
   <property name="icon">
    <iconset theme="edit-select-all" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/edit-select-all.png</normaloff>:/icons/oxygen/32x32/actions/edit-select-all.png</iconset>
   </property>
   <property name="text">
    <string>Select A&amp;ll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFormatBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="format-text-bold" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-text-bold.png</normaloff>:/icons/oxygen/32x32/actions/format-text-bold.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionFormatItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="format-text-italic" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-text-italic.png</normaloff>:/icons/oxygen/32x32/actions/format-text-italic.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionFormatUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="format-text-underline" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-text-underline.png</normaloff>:/icons/oxygen/32x32/actions/format-text-underline.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionFormatStrikethrough">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="format-text-strikethrough" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-text-strikethrough.png</normaloff>:/icons/oxygen/32x32/actions/format-text-strikethrough.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Strikethrough</string>
   </property>
  </action>
  <action name="actionFormatAlignLeft">
   <property name="icon">
    <iconset theme="format-justify-left" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-justify-left.png</normaloff>:/icons/oxygen/32x32/actions/format-justify-left.png</iconset>
   </property>
   <property name="text">
    <string>Align &amp;Left</string>
   </property>
  </action>
  <action name="actionFormatAlignCenter">
   <property name="icon">
    <iconset theme="format-justify-center" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-justify-center.png</normaloff>:/icons/oxygen/32x32/actions/format-justify-center.png</iconset>
   </property>
   <property name="text">
    <string>Align &amp;Center</string>
   </property>
  </action>
  <action name="actionFormatAlignRight">
   <property name="icon">
    <iconset theme="format-justify-right" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-justify-right.png</normaloff>:/icons/oxygen/32x32/actions/format-justify-right.png</iconset>
   </property>
   <property name="text">
    <string>Align &amp;Right</string>
   </property>
  </action>
  <action name="actionFormatAlignJustify">
   <property name="icon">
    <iconset theme="format-justify-fill" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-justify-fill.png</normaloff>:/icons/oxygen/32x32/actions/format-justify-fill.png</iconset>
   </property>
   <property name="text">
    <string>Align &amp;Justify</string>
   </property>
  </action>
  <action name="actionFormatIncreaseIndent">
   <property name="icon">
    <iconset theme="format-indent-more" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-indent-more.png</normaloff>:/icons/oxygen/32x32/actions/format-indent-more.png</iconset>
   </property>
   <property name="text">
    <string>I&amp;ncrease Indent</string>
   </property>
  </action>
  <action name="actionFormatDecreaseIndent">
   <property name="icon">
    <iconset theme="format-indent-less" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-indent-less.png</normaloff>:/icons/oxygen/32x32/actions/format-indent-less.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Decrease Indent</string>
   </property>
  </action>
  <action name="actionFormatBulletedList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="format-list-unordered" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-list-unordered.png</normaloff>:/icons/oxygen/32x32/actions/format-list-unordered.png</iconset>
   </property>
   <property name="text">
    <string>Bulle&amp;ted List</string>
   </property>
  </action>
  <action name="actionFormatNumberedList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="format-list-ordered" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/format-list-unordered.png</normaloff>:/icons/oxygen/32x32/actions/format-list-unordered.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Numbered List</string>
   </property>
  </action>
  <action name="actionInsertImage">
   <property name="icon">
    <iconset theme="insert-image" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/insert-image.png</normaloff>:/icons/oxygen/32x32/actions/insert-image.png</iconset>
   </property>
   <property name="text">
    <string>Insert &amp;Image...</string>
   </property>
  </action>
  <action name="actionCreateLink">
   <property name="icon">
    <iconset theme="insert-link" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/insert-link.png</normaloff>:/icons/oxygen/32x32/actions/insert-link.png</iconset>
   </property>
   <property name="text">
    <string>Create Link...</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset theme="zoom-out" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/zoom-out.png</normaloff>:/icons/oxygen/32x32/actions/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset theme="zoom-in" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/zoom-in.png</normaloff>:/icons/oxygen/32x32/actions/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="window-close" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/window-close.png</normaloff>:/icons/oxygen/32x32/actions/window-close.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lose Window</string>
   </property>
   <property name="toolTip">
    <string>Close Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionStyleParagraph">
   <property name="text">
    <string>&amp;Paragraph</string>
   </property>
  </action>
  <action name="actionStyleHeading1">
   <property name="text">
    <string>Heading &amp;1</string>
   </property>
  </action>
  <action name="actionStyleHeading2">
   <property name="text">
    <string>Heading &amp;2</string>
   </property>
  </action>
  <action name="actionStyleHeading3">
   <property name="text">
    <string>Heading &amp;3</string>
   </property>
  </action>
  <action name="actionStyleHeading4">
   <property name="text">
    <string>Heading &amp;4</string>
   </property>
  </action>
  <action name="actionStyleHeading5">
   <property name="text">
    <string>Heading &amp;5</string>
   </property>
  </action>
  <action name="actionStyleHeading6">
   <property name="text">
    <string>Heading &amp;6</string>
   </property>
  </action>
  <action name="actionStylePreformatted">
   <property name="text">
    <string>Pre&amp;formatted</string>
   </property>
  </action>
  <action name="actionStyleAddress">
   <property name="text">
    <string>&amp;Address</string>
   </property>
  </action>
  <action name="actionFormatFontName">
   <property name="text">
    <string>&amp;Font Name...</string>
   </property>
  </action>
  <action name="actionFormatTextColor">
   <property name="text">
    <string>Text &amp;Color...</string>
   </property>
  </action>
  <action name="actionFormatBackgroundColor">
   <property name="text">
    <string>Bac&amp;kground Color...</string>
   </property>
  </action>
  <action name="actionFormatFontSize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Font Si&amp;ze...</string>
   </property>
  </action>
  <action name="actionInsertHtml">
   <property name="icon">
    <iconset theme="insert-text" resource="../../icons/resources.qrc">
     <normaloff>:/icons/oxygen/32x32/actions/insert-text.png</normaloff>:/icons/oxygen/32x32/actions/insert-text.png</iconset>
   </property>
   <property name="text">
    <string>Insert HTML...</string>
   </property>
   <property name="toolTip">
    <string>Insert HTML</string>
   </property>
  </action>
  <action name="actionTextOutline">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Text Outline...</string>
   </property>
  </action>
  <action name="actionTextShadow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Text Shadow...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
